core/models.py

```diff
@@ -130,6 +130,7 @@ class Languages(models.TextChoices):
     activated = models.BooleanField(null=True, default=False)
     deleted = models.BooleanField(default=False)
     bundle_analysis_enabled = models.BooleanField(default=False, null=True)
+    coverage_enabled = models.BooleanField(default=False, null=True)
```

Comment:
So this won't have a DB default right, is that what you intend?

Reply:
Yeah, only ORM default, just to keep it consistent with the related fields (bundle_analysis_enabled, active, activated).